docs/advanced/hep_examples.rst

```diff
@@ -171,7 +171,68 @@ If you do not want to show and take into account the MC uncertainties, setting `
 
 
 
-For other type of comparisons, see :ref:`basics-1d_hist_comparison-label`.
+Other comparisons
+-----------------
+
+Every type of comparisons available with ``plot_comparison()`` are available for ``compare_data_mc()`` (see :ref:`basics-1d_hist_comparison-label`).
+
+Example plot with all comparisons, using the same histograms as above:
+
+.. code-block:: python
+    from plothist import (
+        create_comparison_figure,
+        compare_data_mc,
+        add_text,
+        set_fitting_ylabel_fontsize
+    )
+    import matplotlib.pyplot as plt
+
+    fig, axes = create_comparison_figure(
+        figsize=(6, 11),
+        nrows=5,
+        gridspec_kw={"height_ratios": [3.3, 1, 1, 1, 1]},
+        hspace=0.3,
+    )
+
+    fig_temp, ax_temp = plt.subplots()
+
+    for k_comp, comparison in enumerate(["ratio", "pull", "relative_difference", "difference"], 1):
```

Comment:
since this figure helped us a lot to catch a bug, I would add two other comparisons: ratio and relative difference with uncorrelated uncertainty instead of split.